<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBoundAco</class>
 <widget class="QDialog" name="AddBoundAco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bounding</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>183</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>高斯</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>白噪声</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>单频声源</string>
      </property>
     </item>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>271</width>
       <height>80</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/new/images/formula1.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>8</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>8</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>T'</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/new/images/formula2.png</pixmap>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>122</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/new/images/formula3.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>8</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>4</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>ω</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>12</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>p</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
   <widget class="QWidget" name="page_6"/>
   <widget class="QWidget" name="page_7"/>
   <widget class="QWidget" name="page_8"/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>52</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>52</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>压力边界</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>绝对软边界</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>绝对硬边界</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>透射边界</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>耦合边界</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>89</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>位置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>89</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>zmin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>xmin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ymin</string>
    </property>
   </item>
  </widget>
  <zorder>label</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_2</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>buttonBox</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddBoundAco</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddBoundAco</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_3</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget_2</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
